Tasks/DockerV0/container.ts

```diff
@@ -61,7 +61,8 @@ else {
 var result = "";
 var telemetry = {
     registryType: containerRegistryType,
-    command: command
+    command: command,
+    jobId: tl.getVariable('SYSTEM_JOBID')
```

Comment:
How will this help?

Reply:
we are recording jobid in processmonitor library. To get the difference between the records of processmonitor library and taskendpointid we need the jobid to be added in the taskendpoint logs. as currently there is no common property to compare the records.